mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileFragment.java

```diff
@@ -144,6 +144,8 @@ public void showProfile(Client client) {
                 .width((int) getResources().getDimension(R.dimen.user_profile_image_size))
                 .height((int) getResources().getDimension(R.dimen.user_profile_image_size))
                 .endConfig().buildRound(client.getName().substring(0, 1), R.color.colorAccentBlack);
+        ImageView editableImageHint=getView().findViewById(R.id.iv_editable_image_hint);
```

Comment:
Space before and after `=` is missing, this will lead to failure in lint checks

Reply:
@kartikeytyagi15 are u still able to added these mention changes in your PR's commit

Reply:
@kartikeytyagi15, @EGOR-IND  I think u guys are not able to solves these changes a this time, So Now I am generated new Pr for this particular issue.